packages/app/src/index.css

```diff
@@ -601,10 +601,6 @@ button.tall {
   justify-content: space-between;
 }
 
-.action-heading button {
-  width: 98px;
```

Comment:
Maybe use `min-width` instead, the fixed width is important

Reply:
done